### PR TITLE
Higher concurrency limit for gunicorn

### PR DESCRIPTION
**Pull request by yunfeng-scale**
Gunicorn returns 503 instead of 429 when concurrency exceeds the limit, before adding rate limiting just increase the concurrency and see if that's good enough

### CODE REVIEW

**Review by ian-scale — COMMENTED** [review]
is there any downside to increasing concurrency by such a large amount?

**Comment by yunfeng-scale** [comment]
> is there any downside to increasing concurrency by such a large amount?

process might OOM, but there's an easy fix, just increase memory

**Comment by ian-scale** [comment]
> > is there any downside to increasing concurrency by such a large amount?
> 
> process might OOM, but there's an easy fix, just increase memory

okay sounds good, approved!

**Review by ian-scale — APPROVED** [review]

File: model-engine/model_engine_server/api/worker.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 from uvicorn.workers import UvicornWorker
 
-# The target concurrency is around 50, so we set the limit to 32 with 4 workers
-# for a total concurrency of 128 to allow for some headroom.
-CONCURRENCY_LIMIT = 32
+CONCURRENCY_LIMIT = 1000
```
Comment (yixu34) [inline model-engine/model_engine_server/api/worker.py:3]: I would add a comment here, basically mirroring the PR description. If I saw this I would be pretty confused without context.

**Pull request merged**